### PR TITLE
Add itslive

### LINKED ISSUE #71: Add ITS_LIVE python library
ITS_LIVE just released a new Python library to get point time series of all land ice, an example notebook can be viewed here 
https://nbviewer.org/github/nasa-jpl/itslive-py/blob/main/notebooks/introduction.ipynb

The library is available via pip or conda

`- itslive>=0.2.0`

### PR DESCRIPTION
**Pull request by weiji14**
Python client for ITSLIVE glacier velocity data! Repo at https://github.com/nasa-jpl/itslive-py

Closes #71, cc @betolink

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ dependencies:
   - networkx==2.8.8
   - numba==0.56.4 # high-performance numerics
   - numpy==1.23.5
-  - pandas==1.5.2
+  - pandas~=1.5.1
```
Comment (weiji14) [inline environment.yml:51]: @betolink, there's a strict pinning on itslive to `pandas==1.5.1` at https://github.com/nasa-jpl/itslive-py/blob/2672896a1c09b21c00a11b587c0d558372b6aaf9/pyproject.toml#L20. Is that intended, or should it be `pandas>=1.5.1` instead?

**Comment by weiji14** [comment]
Will merge this in once the tests pass, let me know @betolink once you've updated itslive-py to a new version and I can push a patch update.

**Comment by betolink** [comment]
Thanks @weiji14 ! I think the pining was just to keep the dependencies in sync but yes it should be `>=` I'll be working on a new version for next week that should come with the patch. I wonder this is the second library where I change the original name and looks like that breaks the conda-forge bots, I didn;t see the automatic PRs. Next time I should stick to the original name

**Pull request merged**